README.md

```diff
@@ -1,5 +1,13 @@
 # bonjour-reflector
 
+## About this project
+
+This projects aims to make bonjour devices (such as printers, chromecasts, ...) discoverable and usable by other devices located in different VLANs.
```

Comment:
I'd add something about how this is similar to avahi-reflector, but in addition permits a fine-grained control of how Bonjour traffic is reflected between VLANs.

Reply:
And this project* typo